src/types.ts

```diff
@@ -780,11 +806,13 @@ export type Filter<T> =
       | FilterOperators<KeyWithSubKeys<T, Key>>;
   };
 
+/** The filter used when updating a document */
 export type UpdateFilter<T> =
   & NotImplementedOperators<"$addToSet">
   & UpdateOperators<T>
   & Partial<T>;
 
+// TODO: what
```

Comment:
Maybe add a comment explaining what Mongo Aggregation Pipelines are?

Reply:
added in latest commit